apps/mobile/src/components/FilterOffersScreen/atom.ts

```diff
@@ -36,10 +37,29 @@ export const locationStateAtom = atom<LocationState | undefined>(
   offersFilterInitialState.locationState
 )
 
-export const locationAtom = atom<Location[] | undefined>(
+export const locationAtom = atom<OfferLocation | undefined>(
   offersFilterInitialState.location
 )
 
+export const locationArrayOfOneAtom = atom(
```

Comment:
Is this the best way? Should the component be made to accept both array and single value atom?